src/main/java/com/jcabi/github/RtRepo.java

```diff
@@ -48,7 +48,8 @@
  * @checkstyle ClassFanOutComplexity (10 lines)
  * @todo #923 Let's implement RtRepo.languages().
  *  Now it returns empty iterable but should return list of repo languages.
- *  Uncomment test RtRepoTest.iteratesLanguages() when done.
```

Comment:
Don't change any puzzles that didn't trigger the current ticket. Otherwise, the tasks that have been created by these other tickets will be closed. Better to create a new puzzle instead.